### DISCUSSION TITLE
用户页面的信息管理不太方便(The management of the user's information is not very convenient)

### DISCUSSION THREAD

**Discussion by cutelolly** [community/community#50547] [Mobile]
虽然github的用户相对较为稳定，有些存储库也不应当轻易更换名称，但是对于手机版本的用户页面，可以稍微添加一些可以自由更换头像，更换主题等等让APP美观度更高的功能，用以减缓审美疲劳。

Although the users of GitHub are relatively stable, some repositories should not easily change the name, but for the mobile version of the user page, you can slightly add some functions that can freely change the avatar, change the theme, etc. to make the APP more aesthetically pleasing to alleviate aesthetic fatigue

**Comment by aconsuegra** [comment]
Thanks for your feedback and the feature request @cutelolly! I'll make sure we keep track of it.
